File: lib/src/main/java/com/sickfutre/android/util/DisplayUtil.java
package com.sickfutre.android.util;

import android.content.Context;
import android.content.res.Resources;
import android.util.TypedValue;

/**
 * Created by Alex Dzeshko on 20-Mar-16.
 */
public class DisplayUtil {

    public static int dpToPx(Context context, int dp) {
        Resources r = context.getResources();
        float px = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, dp, r.getDisplayMetrics());
        return (int) px;
    }
}

File: lib/src/main/java/com/sickfutre/android/util/DisplayUtil.kt
package com.sickfutre.android.util

import android.content.Context
import android.util.TypedValue

/**
 * Created by Alex Dzeshko on 20-Mar-16.
 */
object DisplayUtil {

    fun dp(context: Context, dp: Int): Int {
        val r = context.resources
        val px = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, dp.toFloat(), r.displayMetrics)
        return px.toInt()
    }
}
